com.scoutshonour.Digital.json

```diff
@@ -0,0 +1,53 @@
+{
+  "app-id": "com.scoutshonour.Digital",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "Digital.sh",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--filesystem=~/.renpy:create"
+  ],
+  "build-options": {
+    "env": {
+      "PREFIX": "/app"
+    }
+  },
+  "modules": [
+    {
+      "name": "digital",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cp -R common /app/common",
+        "cp -R game /app/game",
+        "cp -R lib /app/lib",
+        "cp -R renpy /app/renpy",
+        "install Digital.py /app/Digital.py",
+        "install -D Digital-launcher.sh /app/bin/Digital.sh",
+        "install -D README.html /app/README.html",
+        "install -Dm644 com.scoutshonour.Digital.appdata.xml /app/share/appdata/com.scoutshonour.Digital.appdata.xml",
+        "install -Dm644 com.scoutshonour.Digital.desktop /app/templates/com.scoutshonour.Digital.desktop"
```

Comment:
As a sidenote, after just changing the desktop file to be saved in the right directory and building, when I try to run build-export I get a segfault.

[mateus@mateus-laptop flathub]$ LANG=C flatpak build-export repo digital-test
WARNING: Can't find Exec key in /home/mateus/Projects/Flatpak/flathub/digital-test/export/share/applications/com.scoutshonour.Digital.desktop: Key file does not have key ?Exec? in group ?Desktop Entry?

(flatpak build-export:2720): GLib-CRITICAL **: g_strsplit: assertion 'string != NULL' failed
Falha de segmentação (imagem do núcleo gravada)

While, I really did forget to fill the Exec (Because I didn't know yet what I was gonna put there), flatpak could do a little better to deal with this error.